pom.xml

```diff
@@ -7,7 +7,7 @@
         <!-- License file to be used by the com.mycila.license-maven-plugin -->
         <copyrightfile>COPYRIGHT</copyrightfile>
         <!-- Version used by all submodules which is managed by the flatten-maven-plugin -->
-        <global.version>4.4.2</global.version>
+        <global.version>4.5.1</global.version>
```

Comment:
Why not 4.5.0?

Reply:
Hmmmm, good point - will change to 4.5.0